types/simple.test.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 
 import {
   Alert,
+  Accordion,
```

Comment:
This line is throwing the following error
```
ERROR: 5:3   expect     TypeScript@next compile error: 
Module '"."' has no exported member 'Accordion'.
```
I feel like I'm missing something obvious here, I'm not sure what it is though...

Reply:
Try to add it also there: https://github.com/react-bootstrap/react-bootstrap/blob/fbc26bbb817629aeb8a6be953603e0e26e88189b/types/components/index.d.ts

Then it should be available for importing.